toptal/picasso
# feat: support wide desktop screen

[FX-751](https://toptal-core.atlassian.net/browse/FX-751)

### Description

Add support for wide screen desktop (1440px)

p/s: Should `fullWidth` props remains for backward compatibility?

### How to test

- Go to `Page` examples
- Check the example `Wide width` right below the `Default` one

### Review
